src/python/bot/tasks/minimize_task.py

```diff
@@ -1348,6 +1348,31 @@ def do_libfuzzer_cleanse(testcase, testcase_file_path, expected_crash_state):
   return output_file_path
 
 
+def do_antlr_tokenized_minimization(test_function, get_temp_file, data,
```

Comment:
This function is unused and should be added in next cl.

Reply:
Done